sysbuild/Kconfig.approtect

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+choice APPROTECT_HANDLING
+	bool "APPROTECT handling"
+	depends on SOC_SERIES_NRF52X || SOC_NRF5340_CPUNET || \
+		   SOC_NRF5340_CPUAPP || SOC_SERIES_NRF91X
```

Comment:
Maybe `SOC_NRF5340_CPUNET || SOC_NRF5340_CPUAPP` could be simplified to `SOC_SERIES_NRF53X`?